src/language-service/src/schemas/ui-lovelace.ts

```diff
@@ -552,6 +553,16 @@ export interface CastEntityConfig {
   hide_if_unavailable?: boolean;
 }
 
+export interface ButtonEntityConfig {
+  type: "button";
+  name: string;
```

Comment:
Name is optional?

```suggestion
  name?: string;
```

Reply:
Maybe I interpret this wrong, but isnt name required?
See this line:
https://github.com/home-assistant/frontend/blob/96110637d935adc1c6278cc50319a6abc43eeb67/src/panels/lovelace/special-rows/hui-button-row.ts#L32

Reply:
Nevermind me, you are right. I was mixing up the button card with the entity row button. 🤒